extensions/amp-anim/0.1/amp-anim.js

```diff
@@ -105,7 +105,13 @@ class AmpAnim extends AMP.BaseElement {
       return Promise.resolve();
     }
     this.img_.setAttribute('src', src);
-    this.loadPromise_ = loadPromise(this.img_);
+    this.loadPromise_ = loadPromise(this.img_)
+        .catch(error => {
+          if (this.img_.getAttribute('src')) {
```

Comment:
You mean `!this.img_.getAttribute('src')`, right? If `src` was cleared, means that we expect the error.